Gems/PhysX/Code/Editor/ColliderAssetScaleMode.h

```diff
@@ -8,21 +8,20 @@
 
 #pragma once
 
-#include "ColliderSubComponentMode.h"
+#include <Editor/Source/ComponentModes/PhysXSubComponentModeBase.h>
 #include <AzToolsFramework/Manipulators/ScaleManipulators.h>
 
 namespace PhysX
 {
     /// Sub component mode for modifying the asset scale on a collider in the viewport.
-    class ColliderAssetScaleMode
-        : public PhysX::ColliderSubComponentMode
+    class ColliderAssetScaleMode : public PhysXSubComponentModeBase
     {
     public:
         AZ_CLASS_ALLOCATOR_DECL
 
         ColliderAssetScaleMode();
 
-        // ColliderSubComponentMode ...
+        // PhysXSubComponentModeBase ...
```

Comment:
nit: I won't bother repeating this but the convention seems to be `// PhysXSubComponentModeBase overrides ...` to indicate the grouping of all the overridden functions (this is mentioned here - https://github.com/o3de/sig-core/blob/main/governance/API-Ref-Guidelines-Update.md) - up to you if you'd like to update them but for all new code this ideally is the style to use

Reply:
ya, this was just caught in the find replace on the rename. so they're still the old style

Reply:
Okay cool fair enough, if the additions can have the `overrides ...` style that'd be awesome